luaui/Widgets/gui_options.lua

```diff
@@ -2907,6 +2907,7 @@ function init()
 				  widgetHandler:DisableWidget('Grid menu')
 				  widgetHandler:EnableWidget('Build menu')
 			  end
+			  init()
```

Comment:
What's this for?

Reply:
Reloads the menu so that grid-specific settings get hidden/shown when this toggle is changed, since they have no effect with gridmenu disabled (this functionality already existed, but it did not reload properly) 
![image](https://github.com/beyond-all-reason/Beyond-All-Reason/assets/3493638/0deb844e-7af3-4217-bce4-721183a920e6)